src/components/InputField/InputField.tsx

```diff
@@ -6,6 +6,7 @@ import { CheckboxSelect } from "@atlaskit/select"
 import EditorInput from "./Editor/Editor"
 import Switch from "./Switch/Switch"
 import Checkbox from "./Checkbox/Checkbox"
+import Calender from "./calender/CalenderComponent"
```

Comment:
React components follow Pascal casing convention. Please use Pascal casing for file names, container folder and component names.